tests/ci/integration/run_haproxy_integration.sh

```diff
@@ -25,10 +25,13 @@ export LD_LIBRARY_PATH="${AWS_LC_INSTALL_FOLDER}/lib"
 
 function build_and_test_haproxy() {
   cd ${HAPROXY_SRC}
-  make CC="${CC}" -j ${NUM_CPU_THREADS} TARGET=generic USE_OPENSSL=1 SSL_INC="${AWS_LC_INSTALL_FOLDER}/include" SSL_LIB="${AWS_LC_INSTALL_FOLDER}/lib/"
+  make CC="${CC}" -j ${NUM_CPU_THREADS} TARGET=generic USE_OPENSSL=1 SSL_INC="${AWS_LC_INSTALL_FOLDER}/include" \
+      SSL_LIB="${AWS_LC_INSTALL_FOLDER}/lib/" USE_LUA=1  LUA_LIB_NAME=lua5.4
```

Comment:
it would be nice to add USE_QUIC=1 (not enabled automatically)

Reply:
do we really need `USE_LUA=1  LUA_LIB_NAME=lua5.4` ?
what if we leave only `USE_LUA=1` ?

Reply:
That seems to work, I was running into issues locally with the lua name detection.

Reply:
AWS-LC currently doesn't support HAProxy's use of QUIC as the names of some functions are different. You have if/defs for BoringSSL to address that issue. We are thinking about adding that mapping for greater OpenSSL compatibility in CryptoAlg-1982. Otherwise HAProxy could expand [this BoringSSL macro](https://github.com/haproxy/haproxy/blob/master/include/haproxy/quic_tls-t.h#L28) to include AWS-LC.